pandas/tests/plotting/test_backend.py

```diff
@@ -9,7 +9,7 @@
 import pandas
 
 dummy_backend = types.ModuleType("pandas_dummy_backend")
-dummy_backend.plot = lambda *args, **kwargs: None
+dummy_backend.__setattr__("plot", lambda *args, **kwargs: None)
```

Comment:
use setattr() (or a # type: ignore) see https://github.com/python/mypy/issues/7171#issuecomment-520407743

Reply:
Done! Personally preferred the setattr() way over ignoring a test line here, comment didn't specify why monkey-patching would be bad here - can do it the other way as well if you prefer and we just want to skip the line entirely.

Reply:
Thanks for the update, but can you review the merge. looks like something went awry.

Reply:
Oh jeez. Will do!

Reply:
Alright, Travis CI wanted to run a couple of times but merge should be fine now!